File: src/main/java/com/webside/menu/service/MenuService.java
package com.webside.menu.service;

import java.util.List;

import org.apache.ibatis.annotations.Param;

import com.webside.menu.model.MenuEntity;
import com.webside.util.PageData;
import com.webside.util.PageResult;

public interface MenuService {
	public PageResult<List<MenuEntity>> findPage(PageData pd);
	public void add(MenuEntity menuEntity);
	public MenuEntity findById(Integer id);
	public int update(MenuEntity menuEntity);
	public void delete(int[] ids);
	public List<MenuEntity> findByNameUrl(MenuEntity menuEntity);
}

File: src/main/webapp/resources/js/customer/menu/menuService.js
//服务层
app.service('menuService',function($http){
	this.add=function(menuEntity){
		return $http.post('../../menu/add',menuEntity);
	}
	this.findById=function(id){
		return $http.get('../../../menu/findById?id='+id);
	}
	this.update=function(userEntity){
		return $http.post('../../menu/update',userEntity);
	}
	this.del=function(ids){
		return $http.get('../../menu/del?ids='+ids);
	}
});
